gui/MainWindow.cxx

```diff
@@ -1499,6 +1499,11 @@ MainWindow::MainWindow(QWidget* parent, Qt::WindowFlags flags)
     std::bind(&MainWindowPrivate::handleLogMessage, d, _1, _2, _3, _4);
   kv::kwiver_logger::set_global_callback(cb);
 
+  // Work around issue where "Options" menu will not display on MacOS
+#ifdef __APPLE__
+  d->UI.menuComputeOptions->setTitle("0ptions");
```

Comment:
This is likely to also break the ability to select the menu using the keyboard. If the issue is an exact string match being performed, I would strongly recommend at least *trying* inserting a non-printing character such as a [ZWSP](https://en.wikipedia.org/wiki/Zero-width_space):
```suggestion
  d->UI.menuComputeOptions->setTitle(QStringLiteral(u"Options\u200b"));
```

Other things to try if that doesn't work: stick the ZWSP in the middle rather than the end, or try [WJ](https://en.wikipedia.org/wiki/Word_joiner) (`\u2060`).

Reply:
I tried several such spaces in different positions.  I couldn't get it to work.  I'm really not that worried about the keyboard shortcut on this menu item being broken on Apple.  That's much better than the menu being missing entirely.

Reply:
Sure, but *not* having the visual "glitch" is surely better.
What about:
```suggestion
  d->UI.menuComputeOptions->setTitle(QStringLiteral(u"Opt\u2170ons"));
```

Reply:
I will give it a try this afternoon.

Reply:
This latest suggestion does not work. The sub-menu is not present.

Reply:
So I found another potential solution.
```suggestion
  d->UI.menubar->setNativeMenuBar(false);
```
This moves most of the menu to the TeleSculptor window but it does allow the Options sub-menu to display.
<img width="1437" alt="Screen Shot 2021-09-13 at 12 35 00 PM" src="https://user-images.githubusercontent.com/4336975/133122891-e74f74f1-c3c2-4837-ac44-2535ddff9f61.png">

Reply:
From the QT [documentation](https://doc.qt.io/qt-5/qmenubar.html#nativeMenuBar-prop).

Reply:
Yeah, I've seen that option, but it makes TeleSculptor look like a very non-standard Mac application.  I could be convinced otherwise, but I think I prefer the "0ptions" solution with zero.